src/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>lader board</title>
+</head>
+<body>
+  <header>
+    <div class="container">
+      <h1>lader board</h1>
+    </div>
+  </header>
+  <section>
+    <div class="container">
+      <div class="leader-board">
+        <div class="refresh">
+          <span>Refresh the page</span>
+          <button>refersh</button>
+        </div>
+        <div class="board">
+          <ul>
+            <li class="special">ahmed : 100</li>
+            <li>ahmed : 100</li>
+            <li class="special">ahmed : 100</li>
+            <li>ahmed : 100</li>
+            <li class="special">ahmed : 100</li>
+            <li>ahmed : 100</li>
+            <li class="special">ahmed : 100</li>
+            <li>ahmed : 100</li>
+            <li class="special">ahmed : 100</li>
+            <li>ahmed : 100</li>
+            <li class="special">ahmed : 100</li>
+            <li>ahmed : 100</li>
+            <li class="special">ahmed : 100</li>
+            <li>ahmed : 100</li>
+            <li class="special">ahmed : 100</li>
+            <li>ahmed : 100</li>
+            <li class="special">ahmed : 100</li>
+            <li>ahmed : 100</li>
+            <li class="special">ahmed : 100</li>
+            <li>ahmed : 100</li>
+            <li class="special">ahmed : 100</li>
+            <li>ahmed : 100</li>
+            <li class="special">ahmed : 100</li>
+            <li>ahmed : 100</li>
+          </ul>
+        </div>
+      </div>
+      <div class="add-score">
+        <h2>Add your score</h2>
+        <form action="">
+          <label for="name" hidden> add your name</label>
+          <input type="text" id="name" placeholder="name">
+          <label for="score" hidden> add your score</label>
+          <input type="text" id="score" placeholder="score">
+          <button type="submit">submit</button>
+        </form>
+      </div>
+    </div>
```

Comment:
- [ ] Please make sure there's no top margin in this section and the submit button should align to the right side of the screen to match the wireframe design.

### Your design
![image](https://user-images.githubusercontent.com/83803414/201979024-1a8c3810-acf7-44dd-b230-9a01d3424c6d.png)

### Wireframe design
![image](https://user-images.githubusercontent.com/83803414/201979076-8154ec2d-459c-4834-a720-5337f254be85.png)